### PR TITLE
MapImageLayer tables

### PR DESCRIPTION
**Pull request by ThadT**
Sample shows how to query non-spatial tables from a MapImageLayer and query related features.

@saratchandrakarumuri - can you please test on all platforms and ensure it works (100.3).
@nCastle1 - please review code.
--Thanks!

### CODE REVIEW

File: src/Forms/UWP/App.xaml.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
 #if DEBUG
             if (System.Diagnostics.Debugger.IsAttached)
             {
-                DebugSettings.EnableFrameRateCounter = true;
+               // DebugSettings.EnableFrameRateCounter = true;
```
Comment (nCastle1) [inline src/Forms/UWP/App.xaml.cs]: Was this intentional?

File: src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/MapImageLayerTables/MapImageLayerTables.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+            // Get the selected comment feature. If there is no selection, return.
+            ArcGISFeature selectedComment = e.AddedItems[0] as ArcGISFeature;
+            if (selectedComment == null) { return; }
+
+            // Get the map image layer that contains the service request sublayer and the service request comments table.
+            ArcGISMapImageLayer serviceRequestsMapImageLayer = MyMapView.Map.OperationalLayers[0] as ArcGISMapImageLayer;
+
+            // Get the service requests sublayer.
+            ArcGISMapImageSublayer requestsSublayer = serviceRequestsMapImageLayer.Sublayers[0] as ArcGISMapImageSublayer;
```
Comment (nCastle1) [inline src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/MapImageLayerTables/MapImageLayerTables.xaml.cs]: It looks like this variable was never used (applies to all platforms).

File: src/iOS/Xamarin.iOS/ArcGISRuntime.Xamarin.Samples.iOS.csproj [hunk 3]
```diff
@@ -127,44 +127,29 @@
   </PropertyGroup>
   <!-- References -->
   <ItemGroup>
-    <Reference Include="Esri.ArcGISRuntime, Version=100.3.0.0, Culture=neutral, PublicKeyToken=8fc3cc631e44ad86, processorArchitecture=MSIL">
```
Comment (nCastle1) [inline src/iOS/Xamarin.iOS/ArcGISRuntime.Xamarin.Samples.iOS.csproj]: It looks like more daily references got in.

File: src/iOS/Xamarin.iOS/packages.config [hunk 4]
```diff
@@ -1,13 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Esri.ArcGISRuntime.Hydrography" version="100.3.0" targetFramework="xamarinios10" />
-  <package id="Esri.ArcGISRuntime.Xamarin.iOS" version="100.3.0" targetFramework="xamarinios10" />
+  <package id="Esri.ArcGISRuntime.Hydrography" version="100.3.0-daily2052" targetFramework="xamarinios10" />
+  <package id="Esri.ArcGISRuntime.Xamarin.iOS" version="100.3.0-daily2052" targetFramework="xamarinios10" />
```
Comment (nCastle1) [inline src/iOS/Xamarin.iOS/packages.config]: More package changes.

File: src/Forms/UWP/App.xaml.cs (hunk 1 above)
Comment (ThadT) [reply src/Forms/UWP/App.xaml.cs]: (I did that to take the screenshot, forgot to uncomment after).

**Comment by ThadT** [comment]
@saratchandrakarumuri and @nCastle1 please re-review when you can. I believe I've addressed all comments. Thanks!

**Comment by ThadT** [comment]
Thanks @saratchandrakarumuri, I thought I fixed that on iOS/Android. I think the problem was the list would expand to show all items (when I first created the sample, there were only a few, more were added later). I set an explicit height on the control, which (I thought) fixed the issue. Thanks for testing that again, I'll see if I can reproduce/fix.

**Comment by ThadT** [comment]
querysublayertables_android

@saratchandrakarumuri Here's what I see for Android (on emulator or device).

**Comment by saratchandrakarumuri** [comment]
I have tried on couple of device and this is what I see

**Tablet**
screenshot_20180608-152805

**Samsung S7**
screenshot_20180608-152427

@ThadT can you check whether there are any local changes on your machine that makes it work fine locally?

**Comment by nCastle1** [comment]
@ThadT I just tested on device and everything looks correct. 

@saratchandrakarumuri Can you confirm which versions of Android are running on those devices and which version of the Android/Xamarin SDKs you're building with?

**Comment by ThadT** [comment]
@saratchandrakarumuri - I made some minor edits to the layout file for the sample. Can you please test on the same Android devices? Thanks!

**Comment by ThadT** [comment]
UI problems on Xamarin.Android version of the sample were due to an older version of Xamarin (8.1.15). Updating to version 8.3.3 solved the issue.

**Pull request merged**